src/libs/ajax/Catalog.ts

```diff
@@ -295,6 +295,7 @@ export interface DatasetMetadata {
   'dct:title': string;
   'dct:description': string;
   'dct:creator': string;
+  'dct:identifier'?: string;
```

Comment:
It looks like this field is present for snapshot based datasets, but not workspace based datasets.

Reply:
Yes, only snapshot datasets have a `dct:identifier`. Could you add a comment here? I believe we thought we'd get the snapshot ID out of the API but haven't worked on it since it was released.

Reply:
Added in https://github.com/DataBiosphere/terra-ui/pull/4293/commits/e525a3380e3180cafa780c1641622a7d250d9abc.